ocs_ci/ocs/utils.py

```diff
@@ -720,6 +720,12 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True):
         ocs (bool): Whether to gather OCS logs
 
     """
+    if not ('KUBECONFIG' in os.environ
```

Comment:
If we are breaking into multiple lines the parenthesis should be on their own lines.

```python
if not (
    'KUBECONFIG' in os.environ
    or os.path.exists(os.path.expanduser('~/.kube/config'))
):
```

same for the log statement
```python
log.warn(
    "Cannot find $KUBECONFIG or ~/.kubeconfig/config; "
    "skipping log collection"
)
```